docs/CAPABILITIES-BUILT-IN.md

```diff
@@ -25,3 +25,5 @@ might define others.
   `1.4.0` or higher support this.
 - `cosmwasm_2_0` enables `CosmosMsg::Any` and `QueryRequest::Grpc`. Only chains
   running CosmWasm `2.0.0` or higher support this.
+- `cosmwasm_2_1` enables secp256r1 and BLS12-381 signature verification. Only
+  chains running CosmWasm `2.1.0` or higher support this.
```

Comment:
This is checked independent of capabilities. The moment the Wasm has the new required imports the check_wasm step fails, no matter which capabilities wasmd offers. So this explanation should really be about new message support instead of Wasm imports.

Reply:
Ah, good point. I wanted to add that in the Async ack PR, but will add here.